.docs/Scheduling-Windows.md

```diff
@@ -38,9 +38,11 @@ exit
 - `dirhere` with the export directory (e.g. C:\Users\User\Documents\Exports)
 - `formathere` with one of the available export formats
 
-Make sure not to delete the quotes (")
+    Make sure not to delete the quotes (")
```

Comment:
Why the 4 spaces here?

Reply:
Removed them, probably did that instinctively to put that line on the same indentation level as the stuff above it